src/app/commands/cmd_export_sprite_sheet.cpp

```diff
@@ -181,6 +181,7 @@ struct ExportSpriteSheetParams : public NewParams {
   Param<int> shapePadding { this, 0, "shapePadding" };
   Param<int> innerPadding { this, 0, "innerPadding" };
   Param<bool> trim { this, false, "trim" };
+  Param<bool> grid { this, false, "grid" };
```

Comment:
This should be called:
```suggestion
  Param<bool> trimByGrid { this, false, "trimByGrid" };
```
Also I'm not sure if the `trim` parameter should be enabled automatically if `trimByGrid` is specified.

Reply:
Done